stdlib/contrib/bonitoo-io/array/array.flux

```diff
@@ -0,0 +1,106 @@
+// Package array provides functions for interacting with arrays.
+package array
+
+
+import ejson "experimental/json"
+import "json"
+
+// append is a function that appends value to array.
+//
+// ## Parameters
+// - `arr` is the array to operate on.
+// - `v` is the value to append to the array.
+//
+//   Array may be empty. If not, value type must match the type of existing element(s).
+//
+// ## Append to array
+//
+// ```
+// import "contrib/bonitoo-io/array"
+//
+// good = ["foo", "bar"]
+//
+// from(bucket: "my-bucket")
+//   |> range(start: -1h)
+//   |> keep(columns: array.append(arr: ["_time", "_value"], v: good)
+// ```
+builtin append : (arr: [A], v: [A]) => [A]
```

Comment:
Should we call this `concat` instead of append as its really creating a new array that is the concatenation of two other arrays?

Reply:
Done, thank you.

At the first attempt I wanted to implemented as _append_, but then I changed my mind and decided to keep it coherent with immutability of variables in Flux, but left the name of the function unchanged.